addons/mail/models/mail_message_reaction.py

```diff
@@ -32,9 +32,9 @@ def _to_store(self, store: Store, fields):
             data = {
                 "content": content,
                 "count": len(reactions),
-                "guests": Store.Many(reactions.guest_id, ["avatar_128", "name"]),
+                "guest_ids": Store.Many(reactions.guest_id, ["avatar_128", "name"]),
```

Comment:
Don't rename this, it's not a real field on the model it doesn't need this syntax